content/blog/2023/08/24/2023-08-24-gsoc-docker-based-quickstart.adoc

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Docker-based Jenkins quick start examples"
+:tags:
+- gsoc
+- gsoc2023
+- docker
+- docker-compose
+- gitpod
+- documentation
+author: ash-sxn
+description: "Google Summer of Code 2023 Docker-based Jenkins quick start examples."
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+== Table of Contents
+
+. <<About the Project>>
+. <<Phase 1>>
+. <<Phase 2>>
+. <<Next Steps>>
+. <<Acknowledgments>>
+. <<Useful Links>>
+
+== About the Project
+
+The goal of my project was to address the problems and complexities users often face when setting up Jenkins with Docker. The current process involves multiple intimidating steps with security risks. My goal was to solve this problem with Docker Compose by creating a Docker Compose file and regularly testing it. Part of the project also involved adding easy-to-understand documentation and tutorials, as well as a one-click Gitpod setup.
+
+What will changed in Jenkins after this project:
+
+* Previously, users had to use six scary commands to set up Jenkins with Docker (take a look link:https://www.jenkins.io/doc/book/installing/docker/#on-macos-and-linux[here]). After this project, users will only need two commands: `git clone URL` and `docker-compose up -d`.
```

Comment:
We might need to screenshot or give snippets because that document/link is a living document. This means the document can be changed later, and the reader will lose the context.